databricks/koalas/series.py

```diff
@@ -4730,6 +4730,31 @@ def asof(self, where):
         result_series.name = self.name
         return result_series
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a Python scalar.
+
+        Returns
+        -------
+        scalar
+            The first element of Series.
+
+        Raises
+        ------
+        ValueError
+            If the data is not length-1.
+
+        Examples
+        --------
+        >>> kser = ks.Series([10])
+        >>> kser.item()
+        10
+        """
+        item_top_two = self[:2]
+        if len(item_top_two) != 1:
+            raise ValueError("can only convert an array of size 1 to a Python scalar")
+        return item_top_two[0]
```

Comment:
Seems like this still runs Spark jobs twice? We should explicitly call `to_pandas()` or `collect()`?

Reply:
Ah, I'll check and fix it. Thanks !

Reply:
Let's be very careful on this @itholic next time. This one line mistake makes the performance two times slower.

Reply:
@HyukjinKwon 
Yeah, I should have considered much more carefully.
Thanks for reminding me once again, @HyukjinKwon ! I must keep that in mind.